examples/example.js

```diff
@@ -171,20 +161,27 @@ async function deleteTask(owner, token, taskId) {
     metadata
   );
 
-  console.log("Delete Task ", taskId, "\n", result);
+  console.log("Response:\n", result);
 }
 
 async function getWallets(owner, token) {
   const metadata = new grpc.Metadata();
   metadata.add("authkey", token);
 
-  const walletsResp  = await asyncRPC(
+  const walletsResp = await asyncRPC(
     client,
     "GetSmartAccountAddress",
-    { },
+    {},
     metadata
   );
 
+  console.log(
+    `Response:\n`,
+    walletsResp
+  );
```

Comment:
What this example needs to show is the original server response, so I intentionlly print it out here. The `{address, salt, factory}` structure looks good!
```
node example.js wallet
Current environment is:  development
Response:
 {
  wallets: [
    {
      address: '0x6C6244dFd5d0bA3230B6600bFA380f0bB4E8AC49',
      salt: '0',
      factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7'
    },
    {
      address: '0x40Baaa1657e7D4cF5d949FCAc0a88cBB4660050e',
      salt: '1',
      factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7'
    },
    {
      address: '0x4604a122a9dd2B04e528C432B72e83a322e017DC',
      salt: '2',
      factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7'
    },
    {
      address: '0x8C5d5187BE3cFFcd0aDBC020FF26Bd17d56F09c7',
      salt: '11',
      factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7'
    }
  ]
}
Fetching balances from RPC provider ...
Listing smart wallet addresses for 0xc60e71bd0f2e6d8832Fea1a2d56091C48493C788 ...
 [
  {
    address: '0x6C6244dFd5d0bA3230B6600bFA380f0bB4E8AC49',
    salt: '0',
    factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7',
    balances: [ '0.2 ETH', '0 ASA' ]
  },
  {
    address: '0x40Baaa1657e7D4cF5d949FCAc0a88cBB4660050e',
    salt: '1',
    factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7',
    balances: [ '0 ETH', '0 ASA' ]
  },
  {
    address: '0x4604a122a9dd2B04e528C432B72e83a322e017DC',
    salt: '2',
    factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7',
    balances: [ '0 ETH', '0 ASA' ]
  },
  {
    address: '0x8C5d5187BE3cFFcd0aDBC020FF26Bd17d56F09c7',
    salt: '11',
    factory: '0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7',
    balances: [ '0 ETH', '0 ASA' ]
  }
]
```